fast_track.ipynb

```diff
@@ -89,7 +87,7 @@
    "id": "cc2b7ef6-e994-499e-beec-bd4fa120cb75",
    "metadata": {},
    "source": [
-    "GRASS GUI has a single window layout by default, but it is also possible to minimize and/or dock/undock the panels. In the right part, you'll find the **data** browser panel which will allow you to navigate through your projects and data, and the **layers** panel showing displayed layers. The panel in the middle is the **map display**. You might add several of those if you need using ![](img/monitor-create.png). Finally, in the right side there's a multiple tab panel where you can find a searchable **tools' tree** similar to the Processing toolbox in QGIS, a **console** where you can type GRASS commands, the **history of executed commands** in case you want to re-run a task and a simple **Python console** where you can use the GRASS Python API. "
+    "GRASS GUI has a single window layout by default, but it is also possible to minimize and/or dock/undock the panels. On the right, you can find the **data** browser which allowa you to navigate through your projects and data, and the **layers** panel showing displayed layers. The panel in the middle is the **map display**. You can add additional ones if you need using ![](img/monitor-create.png). Finally, on the right there are multiple tabs where you can find a searchable **tools' tree** similar to the Processing toolbox in QGIS, a **console** where you can type GRASS commands, the **history of executed commands** in case you want to re-run a task and a simple **Python console** where you can use the GRASS Python API. "
```

Comment:
```suggestion
    "GRASS GUI has a single window layout by default, but it is also possible to minimize and/or dock/undock the panels. On the right, you can find the **data** browser which allows you to navigate through your projects and data, and the **layers** panel showing displayed layers. The panel in the middle is the **map display**. You can add additional ones if you need using ![](img/monitor-create.png). Finally, on the right there are multiple tabs where you can find a searchable **tools' tree** similar to the Processing toolbox in QGIS, a **console** where you can type GRASS commands, the **history of executed commands** in case you want to re-run a task and a simple **Python console** where you can use the GRASS Python API. "
```